gradebookng/tool/src/webapp/styles/gradebook-grades.css

```diff
@@ -321,7 +321,16 @@ div.wicket-modal div.w_caption {
 div.wicket-modal div.w_right > div {
   box-shadow: 0 0 5px 2px #444;
 }
-
+div.wicket-modal[aria-labelledby="Sort Grade Items"] div.w_caption,
+div.wicket-modal[aria-labelledby="Sort Grade Items"] div.w_content,
+div.wicket-modal[aria-labelledby="Bulk Edit Items"] div.w_caption,
+div.wicket-modal[aria-labelledby="Bulk Edit Items"] div.w_content,
+div.wicket-modal[aria-labelledby="Add Gradebook Item"] div.w_caption,
+div.wicket-modal[aria-labelledby="Add Gradebook Item"] div.w_content
+{
+  background: var(--sakai-background-color-1);
+  color: var(--sakai-text-color-1);
+}
```

Comment:
I couldn't figure out another way to target these. The aria-labels don't change with user language preference so it seemed pretty robust. Welcome to other ideas though.

Reply:
As a follow-up to what @profmikegreene @SedueRey @adrianfish @ern @bjones86 and others discussed on the Core call today:

I think those aria-labelledby attributes are actually incorrect in Gradebook. Their value should be an element id that contains the text to describe it by rather than text to describe it. Therefore, their value should be the id of modal's header text, for example. 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-labelledby_attribute

This probably explains why it doesn't change when the user language preference changes - because the descriptive text is hard-coded into those values. 

We'll have to submit a separate Jira issue about that.

Reply:
We decided to make the selector more specific using the Wicket colour class that was already provided.